demos-europe/demosplan-core
# B t17533 adjust print in blueprints using basemap

**Ticket:** https://yaits.demos-deutschland.de/T17533

Description:
1. Update the columns _g_print, default_visibility and enabled to true of _gis table that use basemap and that belong only to masterblueprints.
2. Set _g_print, default_visibility,  enabled to true for the default data whenever a new DemosClient will get demosCore (demosplan/DemosPlanCoreBundle/DataFixtures/ORM/ProdData/LoadProcedureData.php)

### How to review/test
- Create a new Verfahren selecting the "leere Blauepause"
- Once created, go to Planungsdokumente und Planzeichnung > Planzeichenerklärung hochladen 
- Scroll to the bottom to see basemap and the following checkmarks should be checked:
-- Diese Kartenebene wird auch für den Druck von Verortungen in Druckansichten und PDF verwendet
--  Diese Kartenebene beim Laden der Karte anzeigen
--  Kartenebene veröffentlichen 
- Just for information: this tooltip is  that _g_print column that is checked when you click on "bearbeiten"

- [ x] Move the tickets on the board accordingly
